sidebars.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
      ],
   },
 ],
-  "Secure Connections": [
+  "Sauce Trusted Connection": [
```

Comment:
I noticed that all of the pages in `test-results` weren't in `sidebars.js`, so they were not searchable and were islands.